lib/translate/inquiry.cc

```diff
@@ -0,0 +1,298 @@
+// Copyright 2020 Google LLC
```

Comment:
Other translation PRs have been using the naming scheme xxxx_translator.h/cc. We should standardize a naming scheme

Reply:
i vote we put all of the engines under "commands" or "translators" dir but name the engines the command name e.g. `inquiry.cc`